src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -986,6 +986,18 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
 class CephfsMirrorService(CephService):
     TYPE = 'cephfs-mirror'
 
+    def config(self, spec: ServiceSpec) -> None:
+        # make sure mirroring module is enabled
+        mgr_map = self.mgr.get('mgr_map')
+        mod_name = 'mirroring'
+        if mod_name not in mgr_map.get('services', {}):
+            self.mgr.check_mon_command({
+                'prefix': 'mgr module enable',
+                'module':  mod_name
```

Comment:
```suggestion
                'module': mod_name
```

cephadm/services/cephadmservice.py:996:26: E241 multiple spaces after ':'
1     E241 multiple spaces after ':'